lib/chewy/journal.rb

```diff
@@ -87,9 +97,13 @@ def delete!
       def clean_until(time)
         query = {
           query: {
-            range: {
-              created_at: {
-                lte: time.to_i
+            filtered: {
+              query: {
```

Comment:
I meant that it worked when I changed `query` to `filter`, but forget it. It doesn't. It makes my Elastic yellow